sdk/dynatrace/Azure.ResourceManager.DynaTrace/tests/Scenario/MonitorResourceCollectionTests.cs

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.DynaTrace.Tests
+{
+    public class MonitorResourceCollectionTests : MonitorTestBase
+    {
+        public MonitorResourceCollectionTests(bool async)
+            : base(async)//, RecordedTestMode.Record)
+        {
+        }
+
+        [TestCase]
+        [RecordedTest]
+        public async Task CreateOrUpdate()
+        {
+            string monitorName = Recording.GenerateAssetName("testDT-");
+            MonitorResource monitorResource = await CreateMonitorResourceAsync(monitorName);
+            Assert.AreEqual(monitorName, monitorResource.Data.Name);
+        }
+
+        [TestCase]
+        [RecordedTest]
+        public async Task Get()
+        {
+            var collection = await GetMonitorResourceCollectionAsync();
+            string monitorName = Recording.GenerateAssetName("testDT-");
+
+            var input = GetMonitorInput();
```

Comment:
Can we use the unified approach to create the monitor resource? I'd prefer to use the `CreateMonitorResourceAsync` method in all these tests to make it look better.